script/binary

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+if [ -z "$1" ]; then
+    OS_PLATFORM_ARG=(-os="darwin linux windows")
+else
+    OS_PLATFORM_ARG=($1)
+fi
+
+if [ -z "$2" ]; then
+    OS_ARCH_ARG=(-arch="386 amd64 arm")
+else
+    OS_ARCH_ARG=($2)
+fi
+
+# Get rid of existing binaries
+rm -f docker-compose*
+rm -rf Godeps/_workspace/pkg
+
+# Build binaries
+gox "${OS_PLATFORM_ARG[@]}" "${OS_ARCH_ARG[@]}" \
```

Comment:
I used gox because docker-machine used gox :)  (point finger at @ehazlett).  I'd actually like to not use gox because I had issue with statically compiling things.  I'd like the linux binaries to be fully static `-linkmode external -extldflags -static` but then windows below up.  We can switch away from gox